assets/js/modules/reader-revenue-manager/datastore/constants.js

```diff
@@ -20,7 +20,7 @@ export const ERROR_CODE_NON_HTTPS_SITE = 'non_https_site';
 
 export const MODULES_READER_REVENUE_MANAGER = 'modules/reader-revenue-manager';
 
-export const MODULE_SLUG = 'reader-revenue-manager';
+export const READER_REVENUE_MANAGER_MODULE_SLUG = 'reader-revenue-manager';
```

Comment:
This is really a module-level constant, not specific to the datastore so it should be exposed similar to `assets/js/modules/analytics-4/constants.js`